magento/magento2-login-as-customer
# magento/magento2-login-as-customer#180: Deprecated records should be removed from a login_as_customer table by cron.

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2-login-as-customer#180: Deprecated records should be removed from login_as_table by cron.

### Manual testing scenarios (*)

1. Perform Login as Customer action.
2. Make sure some records are present in the login_as_customer table.
3. Run cron.
4. Wait.
5. Make sure no records older than 60 seconds are present in the login_as_customer table.


### Questions or comments


### Contribution checklist (*)
